metadata/metadata.go

```diff
@@ -51,7 +51,7 @@ func DecodeKeyValue(k, v string) (string, string, error) {
 // two convenience functions New and Pairs to generate MD.
 type MD map[string][]string
 
-// New creates a MD from given key-value map.
+// New creates a md from given key-value map.
```

Comment:
Here MD is the correct usage.  It creates something of the *type* MD.

"md" should be used where it's referring to a parameter's name.